### DISCUSSION TITLE
Need help with 2 SMPlayer issues, video stuttering & space bar resume

### DISCUSSION THREAD

**Discussion by butchnn** [smplayer-dev/smplayer#1063] [General]
Hi all, seems my old account isn’t here so I had to make a new one.
I’m not a noob to using SMPlayer.
I have an Alienware M18R1, i9 13900HX, 64 GB DDR5 RAM, NVIDIA GeForce RTX 4090 GPU & 2 WD Black 850 NVMe SSD’s.
My SMPlayer Version is 24.5.0 (revision 10277) (64-bit)

Even though I’ve been using SMPlayer for many years & I am pretty familiar with using it I’ve recently run into a couple of issues.

1st : For some reason while watching videos it will stutter.
2nd  : When using the spacebar to pause I have to press the spacer not once but twice to resume.

I’ve just started experiencing both issue & I’ve changed nothing.
Any insight on both issues would be greatly appreciated.
Thanks!

**Comment by smplayer-dev** [comment]
> 1st : For some reason while watching videos it will stutter. 

Try selecting another video output in preferences -> general -> video.
If that doesn't fix it, there are several option under preferences -> performance you can change. For example try adding more threads for decoding or enable/disable decoding by hardware.

> 2nd : When using the spacebar to pause I have to press the spacer not once but twice to resume.

I think this is fixed in this version:
 https://github.com/smplayer-dev/smplayer/releases/tag/v24.5.0.10323

**Comment by bendover22** [comment]
You can try selecting video output = gpu, or if doesn't help, vo= auto.
Didn't say the OS.  But look at a system monitor app to see what CPU & ram usage is, before / after playing smp.
The overall system resources used / free, & what smp uses.

Same problem if start vid(s) in MPV (w/o using smp)?

Can try a clean, default smplayer.ini to see if altered settings are causing problem (w/ smp closed, BU / rename the current .ini.  Restart smp & it'll generate a default .ini).
Sometimes, GPUs can have problems causing stuttering.
